docs/tools/vdb_table/data/myscale.json

```diff
@@ -132,13 +132,13 @@
     "comment": ""
   },
   "sharding": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://clickhouse.com/docs/en/engines/table-engines/special/distributed",
```

Comment:
Any link for this within your docs? Might be clearer to not send the reader to 3rd party documentation, even if you use the 3rd party under the hood.

If not, then at least let's explain this in the comment: field.